integrationExamples/gpt/blueconicRtdProvider_example.html

```diff
@@ -0,0 +1,123 @@
+<html>
+<head>
+    <script>
+      var testBlueconicData = `{"bc_segments": ["allvisitors"], "bidId": "1234", "identifier": ["test","123"]}`;
+      window.localStorage.setItem('bcPrebidData', testBlueconicData);
+    </script>
+
+    <script>
+        var FAILSAFE_TIMEOUT = 2000;
+
+        var adUnits = [
+            {
+                code: 'test-div',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300,250],[300,600],[728,90]]
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'appnexus',
+                        params: {
+                            placementId: 13144370
+                        }
+                    }
+                ]
+            }
+        ];
+
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+    </script>
+    <script src="../../build/dev/prebid.js" async></script>
```

Comment:
please point to a real location of prebid so integration examples can one day be used for e2e tests

Reply:
Hi @patmmccann 
Can you please tell me whats exact location of prebid and gpt because I can see same relative address in other examples as well ? Is something changed recently regarding this ?

Reply:
Yeah needs some cleanup

See here for prebid https://docs.prebid.org/dev-docs/examples/basic-example.html

See here for gpt

https://developers.google.com/publisher-tag/guides/get-started